figures/models.py

```diff
@@ -294,6 +294,9 @@ class LearnerCourseGradeMetrics(TimeStampedModel):
     points_earned = models.FloatField()
     sections_worked = models.IntegerField()
     sections_possible = models.IntegerField()
+    letter_grade = models.CharField(max_length=255, blank=True, default='')
```

Comment:
The `letter_grade` and `percent_grade` can be changed when a learner re-attempt or complete remaining graded units. @iam-mhaseeb have you verified that these stats will be reflected everytime after our figures data update job is run?

Reply:
@zubair-arbi This is kind of uncertain, running multiple tests I found percent_grade do update but letter_grade remains same.